templates/resources/vm_inserted_media.md.tmpl

```diff
@@ -0,0 +1,24 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: "VM (Virtual Machine)"
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+
+  Note: Only one media is allowed per VM.
```

Comment:
`~>` Add Warning Block in terraform doc. 
Preview result in [doc-preview](https://registry.terraform.io/tools/doc-preview)
```suggestion
 ~> Note: Only one media is allowed per VM.
```